js/bybit.js

```diff
@@ -49,8 +49,9 @@ module.exports = class bybit extends Exchange {
                 'fetchDepositAddresses': false,
                 'fetchDepositAddressesByNetwork': true,
                 'fetchDeposits': true,
-                'fetchFundingRate': true,
+                'fetchFundingRate': true, // emulated in exchange
                 'fetchFundingRateHistory': true,
+                'fetchFundingRates': true,
                 'fetchIndexOHLCV': true,
                 'fetchLedger': true,
                 'fetchMarketLeverageTiers': true,
```

Comment:
don't we have a `fetchBorrowInterest` v5 counterpart?
![image](https://user-images.githubusercontent.com/43336371/220912236-43fe6763-e4f8-4ead-8b29-3008c2d68c7b.png)

Reply:
@carlosmiei it seems they don't have v5 api for this. But I found another api might be for contract https://bybit-exchange.github.io/docs/v5/account/borrow-history